DOCS.md

```diff
@@ -54,19 +54,27 @@ pipeline:
 +   sensitive: true
 ```
 
-Example configuration with state tracked via remote:
+Example configuration with state tracked via remote.  You will need a file
```

Comment:
It does not need to be a separate file. All is needed with TF 0.9.x is the new `terraform` block with a backend definition if remote state is used. It can be added to existing `.tf` file(s).